var/spack/repos/builtin/packages/gaussian-view/package.py

```diff
@@ -13,29 +14,78 @@ class GaussianView(Package):
     Gaussian. It aids in the creation of Gaussian input files, enables the
     user to run Gaussian calculations from a graphical interface without the
     need for using a command line instruction, and helps in the interpretation
-    of Gaussian output"""
+    of Gaussian output.
+    Needs post-install steps to make it run!
+    See package installation logs for details."""
 
     homepage = "https://gaussian.com/gaussview6/"
     manual_download = True
 
-    version('6016',
+    version('6.1.6',
+            sha256='c9824fd0372c27425b53de350f3a83b27de75ca694219b3ef18cd7d92937db6c',
+            extension='tbz')
+
+    version('6.0.16',
             '5dd6a8df8c81763e43a308b3a18d2d3b825d3597e9628dcf43e563d1867b9638',
             extension='tbz')
 
-    depends_on('gaussian@16-B.01', type='run')
+    variant('gaussian-src', default=False,
+            description='Use gaussian-src instead of the prebuild gaussian')
+
+    depends_on('gaussian@16-B.01', type='run', when='@:6.0.99')
+    depends_on('gaussian@16-C.01', type='run', when='~gaussian-src@6.1:')
+    depends_on('gaussian-src@16-C.01', type='run', when='+gaussian-src@6.1:')
+
+    conflicts('+gaussian-src', when='@:6.0.99')
```

Comment:
Does it really conflict? This are two different Gaussian versions. What did I miss?

Reply:
When doing `spack load gaussian-view` we want Gaussian to be loaded automatically, correct?
By adding the `gaussian-src` variant the admin can switch which version of Gaussian is being pulled in the spec.
Since I don't support older versions than `16-C.01` in `gaussian-src` I add a conflict on `+gaussian-src` for older versions of Gaussian View.